experimental/packages/otlp-grpc-exporter-base/src/OTLPGRPCExporterNodeBase.ts

```diff
@@ -126,16 +125,18 @@ export abstract class OTLPGRPCExporterNodeBase<
       return;
     }
 
-    const promise = this._transport.send(data).then(response => {
-      if (response.status === 'success') {
-        onSuccess();
-        return;
-      }
-      if (response.status === 'failure' && response.error) {
-        onError(response.error);
-      }
-      onError(new OTLPExporterError('Export failed with unknown error'));
-    }, onError);
+    const promise = this._transport
+      .send(data, this.timeoutMillis)
+      .then(response => {
+        if (response.status === 'success') {
+          onSuccess();
+          return;
+        }
+        if (response.status === 'failure' && response.error) {
+          onError(response.error);
+        }
+        onError(new OTLPExporterError('Export failed with unknown error'));
```

Comment:
Same Q as before about `onError` possibly being called twice.

Reply:
Also fixed in [5e40f0d](https://github.com/open-telemetry/opentelemetry-js/pull/4889/commits/5e40f0dda6d8572acaf5bd63ad16c92d73ae46f5) :slightly_smiling_face: